source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -2065,6 +2140,31 @@ TEST_F(NiDAQmxDriverApiTests, LoadedVoltageTask_ReadAIData_ReturnsDataInExpected
   EXPECT_DATA_IN_RANGE(read_response.read_array(), AI_MIN, AI_MAX);
 }
 
+TEST_F(NiDAQmxDriverApiTests, AIBridgeChannel_BridgeOffsetNullingCal_ReturnsError)
+{
+  const auto AI_MIN = 0.0;
+  const auto AI_MAX = 1.0;
+
+  EXPECT_THROW_DRIVER_ERROR({
+    auto status = create_ai_bridge_chan(AI_MIN, AI_MAX);
+  }, MEASUREMENT_TYPE_NOT_SUPPORTED_BY_SELECTED_PHYSICAL_CHANNEL_ERROR);
+}
+
+TEST_F(NiDAQmxDriverApiTests, AIThrmcplChannel_ThrmcplLeadOffsetNullingCal_ReturnsError)
+{
+  const auto AI_MIN = 0.0;
+  const auto AI_MAX = 100.0;
+  create_ai_thrmcpl_chan(AI_MIN, AI_MAX);
+
+  std::string channel = "";
+  bool skip_unsupported_channels = false;
+
+  EXPECT_THROW_DRIVER_ERROR({
+    auto response = PerformThrmcplLeadOffsetNullingCalResponse{};
+    auto status = perform_thrmcpl_lead_offset_nulling_cal(channel, skip_unsupported_channels, response);
+  }, THERMOCOUPLE_LEAD_OFFSET_NULLING_CALIBRATION_NOT_SUPPORTED_BY_DEVICE_ERROR);
```

Comment:
Please read https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/20510/Regression-Test-Conventions?anchor=format if you haven't already.

The test body should be split into arrange/act/assert parts separated by a blank line. In this case, act and assert are combined due to the EXPECT_THROW_DRIVER_ERROR macro, so there should be only two parts.

You can merge these two declarations into either the arrange or act part.
```suggestion
  std::string channel = "";
  bool skip_unsupported_channels = false;
  EXPECT_THROW_DRIVER_ERROR({
    auto response = PerformThrmcplLeadOffsetNullingCalResponse{};
    auto status = perform_thrmcpl_lead_offset_nulling_cal(channel, skip_unsupported_channels, response);
  }, THERMOCOUPLE_LEAD_OFFSET_NULLING_CALIBRATION_NOT_SUPPORTED_BY_DEVICE_ERROR);
```